test/Microsoft.AspNetCore.Routing.Tests/RoutingBuilderExtensionsTest.cs

```diff
@@ -6,8 +6,8 @@
 using Microsoft.AspNetCore.Builder.Internal;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Routing;
-using Microsoft.AspNetCore.Routing.Internal;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Internal;
```

Comment:
Could we use `Task.FromResult` in tests? It avoids type conflict issues if we ever add an `InternalsVisibleTo` in the source project.